prac_05/emails.py

```diff
@@ -1,7 +1,30 @@
 
+QUIT = ""
+YES = "Y"
```

Comment:
Consider making YES = ["Y", "YES"] and in line 13, change '!= YES' to 'not in YES'. This is just in case the user enters 'yes' as in input instead of 'y'.